notebooks/Satellite_Data/GOES_Interactive_Plot.ipynb

```diff
@@ -117,12 +117,9 @@
     "               color='white', fontsize='large', weight='bold')\n",
     "\n",
     "    # Make the text stand out even better using matplotlib's path effects\n",
-    "    text_time.set_path_effects([patheffects.Stroke(linewidth=2, foreground='black'),\n",
-    "                           patheffects.Normal()])\n",
-    "    text_channel.set_path_effects([patheffects.Stroke(linewidth=2, foreground='black'),\n",
-    "                           patheffects.Normal()])\n",
-    "    text_experimental.set_path_effects([patheffects.Stroke(linewidth=2, foreground='black'),\n",
-    "                           patheffects.Normal()])\n",
+    "    text_time.set_path_effects([patheffects.withStroke(linewidth=2, foreground='black')])\n",
+    "    text_channel.set_path_effects([patheffects.withStroke(linewidth=2, foreground='black')])\n",
```

Comment:
Good point. Cleans it up a bit. Will do and also copy in the MetPy gallery if we have multiple text placements anywhere there.